### ISSUE TITLE
Fix Configuration File

### ISSUE THREAD

**Issue by kb1lqc** [FaradayRF/clerk#2]
## Summary
Provide a generic configuration file setup for a base install of influxdb settings. Allow user to edit this as needed with command line options.

## Problem Explanation
Installation should be easy and configuration also easy.

# Supporting Information
>This is a handy spot to post any screenshots or file outputs in order to better support this issue. We want to keep the top of the ticket clutter-free as possible.

**Comment by kb1lqc** [comment]
When aprslib is not configured I throw this error:

```
ERROR:aprslib.inet.IS:Failed to login
Traceback (most recent call last):
  File "aprs2influxdb/main.py", line 247, in <module>
    main()
  File "aprs2influxdb/main.py", line 233, in main
    AIS.connect()
  File "C:\Python27\lib\site-packages\aprslib\inet.py", line 115, in connect
    self._send_login()
  File "C:\Python27\lib\site-packages\aprslib\inet.py", line 310, in _send_login
    raise LoginError("Failed to login")
aprslib.exceptions.LoginError: Failed to login
```

**Comment by kb1lqc** [comment]
Just fixed this exception (loginerror) with a try/except statement

**Comment by kb1lqc** [comment]
Updates implemented in latest commits to issue6 branch

- Added support for editing config values through command line
- Removed initialization option
- renamed some configs
- Added some exception handling for aprs connections

**Comment by kb1lqc** [comment]
Largely fixed by #11 though I'm not sure it's even useful to have a configuration file. Might be best to just do all by command line!
